cmd/apiserver/main.go

```diff
@@ -12,91 +12,43 @@ import (
 	logger "github.com/sirupsen/logrus"
 
 	"github.com/pokt-foundation/portal-api-go/repository"
+	"github.com/pokt-foundation/utils-go/environment"
 
 	// TODO: replace with pokt-foundation/relay-meter
 	"github.com/adshmh/meter/api"
-	"github.com/adshmh/meter/cmd"
 	"github.com/adshmh/meter/db"
 )
 
-const (
-	LOAD_INTERVAL_DEFAULT_SECONDS      = 30
-	DAILY_METRICS_TTL_DEFAULT_SECONDS  = 120
-	TODAYS_METRICS_TTL_DEFAULT_SECONDS = 60
-	MAX_ARCHIVE_AGE_DEFAULT_DAYS       = 30
-	SERVER_PORT_DEFAULT                = 9898
-
-	ENV_LOAD_INTERVAL_SECONDS      = "LOAD_INTERVAL_SECONDS"
```

Comment:
IMO these are useful constants that show clearly the expected settings and their default values, rather than allowing them to be hard-coded strings elsewhere.